packages/lodestar/src/chain/chain.ts

```diff
@@ -213,28 +237,8 @@ export class BeaconChain extends EventEmitter implements IBeaconChain {
     if(!isValidGenesisState(this.config, genesisState)) {
       this.logger.info(`Eth1 block ${eth1Block.hash} is NOT forming valid genesis state`);
     }
+    await this.initializeBeaconChain(genesisState, merkleTree);
```

Comment:
semantic, but the log is incorrect, either change the log to `Initialized` as its completed or swap line 240 and 241